runner.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/game'
+require './lib/player'
+require './lib/space'
+require './lib/computer'
+require 'pry'
+
+
+game = Game.new
+
+game.print_welcome
+
+game.print_board
+
+player1 = Player.new(game)
+
+computer = Computer.new(game)
+
+turn_counter = 0
+
+until player1.has_won? do
+
```

Comment:
Great refactor here! The spacing makes the game board a lot easier to read when playing. I also like how it tells you what turn you're currently on.